resources/infrastructure.yaml

```diff
@@ -0,0 +1,34 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: Infrastructure needed by AWS Node Termination Handler.
```

Comment:
Update Descriptions with resources, ex: `IAM Policy detailing the permissions needed by AWS Node Termination Handler`